core/data/src/main/java/org/mifos/mobilewallet/core/domain/usecase/invoice/FetchInvoices.kt

```diff
@@ -0,0 +1,39 @@
+package org.mifos.mobilewallet.core.domain.usecase.invoice
+
+import org.mifos.mobilewallet.core.base.UseCase
+import org.mifos.mobilewallet.core.data.fineract.entity.Invoice
+import org.mifos.mobilewallet.core.data.fineract.repository.FineractRepository
+import rx.Subscriber
+import rx.android.schedulers.AndroidSchedulers
+import rx.schedulers.Schedulers
+import javax.inject.Inject
+
+/**
+ * Created by ankur on 11/June/2018
+ */
+class FetchInvoices @Inject constructor(private val mFineractRepository: FineractRepository) :
+    UseCase<FetchInvoices.RequestValues?, FetchInvoices.ResponseValue?>() {
+
+    class RequestValues(val clientId: String) : UseCase.RequestValues
+    class ResponseValue(
+        val invoiceList: List<Invoice?>
+    ) : UseCase.ResponseValue
+
+    override fun executeUseCase(requestValues: RequestValues?) {
```

Comment:
This should not be nullable, Please remove `?`

Reply:
done